src/main/java/frc/robot/RobotContainer.java

```diff
@@ -9,16 +9,73 @@
 import edu.wpi.first.wpilibj2.command.button.CommandXboxController;
 
 public class RobotContainer {
-  private final CommandXboxController kPilotController =
-      new CommandXboxController(Constants.kPilotControllerPort);
 
-  public RobotContainer() {
-    configureBindings();
-  }
+    private final CommandXboxController driverController = new CommandXboxController(0);
+    private final CommandXboxController operatorController = new CommandXboxController(1);
 
-  private void configureBindings() {}
+    // Define subsystems
+    // ex: private final LEDSubsystem LEDs;
+
+    // Define other utility classes
+    private final AutonCommands autonCommands;
+    private final TeleopCommands telopCommands;
+
+    private final LoggedDashboardChooser<Command> autoChooser;
+
+    private final boolean useCompetitionBindings = true;
+
+    public RobotContainer() {
+
+        // If using AdvantageKit, perform mode-specific instantiation of subsystems.
+        switch (Constants.kCurrentMode) {
+            case REAL:
+                // Instantiate subsystems that operate actual hardware (Hardware controller based modules)
+                break;
+            case SIM:
+                // Instantiate subsystems that simulate actual hardware (IOSim modules)
+                break;
+            default:
+                // Instantiate subsystems that are driven by playback of recorded sessions. (IO modules)
+                break;
+        }
+
+        // Instantiate subsystems that don't care about mode, or are non-AdvantageKit enabled.
+        // ex: LEDs = new LEDSubsystem();
+
+        // Instantiate your TeleopCommands and AutonCommands classes
+        telopCommands = new TeleopCommands(/* pass subsystems here */);
+        autonCommands = new AutonCommands(/* pass subsystems here */);
+        autoChooser = new LoggedDashboardChooser<>("Auton Program", autonCommands.getAutoChooser());
```

Comment:
If we are not using Shuffleboard, and are using Elastic (https://github.com/Gold872/elastic-dashboard) instead, we need to have this verified now in the off-season.  Has anyone used Elastic yet?

Reply:
We should be using a LoggedDashboardChooser, with the CommandUtils buildAutoChooser() method to avoid nulling out the roborio. I will fix that in the next commit.
Jack D. and I have, it's the same process as Shuffleboard but without memory issues and much better UI(Also, it is part of WPILib 2025). The autonomous tab is completely optional if you are not using shuffleboard.

Reply:
I will have to rescind my comment on the LoggedDashboardChooser. Since all the paths are created manually in sequential commands, the CommandUtils.buildAutoChooser() is no longer necessary to generate paths. But the try catch on the creation of the chooser is necessary, to avoid funny network tables failures.